src/wovnio/wovnphp/API.php

```diff
@@ -45,6 +47,7 @@ public static function translate($store, $headers, $original_content, $request_o
             $converted_html = $converter->convertToAppropriateBodyForApi($converted_html);
         }
         $converted_html = $converter->insertSnippetAndHreflangTags($converted_html, true);
+        $converted_html = $converter->insertHtmlLangAttribute($converted_html, $default_lang);
```

Comment:
You might also want to combine this with `insertSnippetAndHreflangTags`

Reply:
`insertWovnSeoTags`, would be my attempt.

Reply:
OK.
How about `InsertSnippetAndLangTags` ?

Reply:
that will do

Reply:
done